examples/js/loaders/GLTFLoader.js

```diff
@@ -1603,6 +1603,8 @@ THREE.GLTFLoader = ( function () {
 
 				materialParams.transparent = true;
 
+				materialParams.alphaTest = material.alphaCutoff || 0.5;
```

Comment:
Could you make this:

```
materialParams.alphaTest = alphaMode === ALPHA_MODES.MASK ? material.alphaCutoff : 0.5;
```

... or an if/else version of that? From the [alpha coverage spec](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#alpha-coverage):

> When alphaMode is set to MASK the alphaCutoff property specifies the cutoff threshold. If the alpha value is greater than or equal to the alphaCutoff value then it is rendered as fully opaque, otherwise, it is rendered as fully transparent. alphaCutoff value is ignored for other modes.